### PR TITLE
Making better EoC doc, part one

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
Making some doc stuff
#### Additional context
Magic of expanding 40 strings into 1000+

### CODE REVIEW

File: doc/EFFECT_ON_CONDITION.md [hunk 1]
```diff
@@ -172,3 +172,1082 @@ Other EOCs have some variables as well that they have access to, they are as fol
...
+
+##### Valid talkers:
+
+| Avatar | Character | NPC | Monster |  Furniture | Item |
+| ------ | --------- | --------- | ---- | ------- | --- | 
+| ✔️ | ✔️ | ✔️ | ❌ | ❌ | ❌ |
+
+##### Examples
+```json
+{ "u_mutate_category": 0 }
```
Comment (andrei8l) [inline doc/EFFECT_ON_CONDITION.md:197]:
```suggestion
{ "u_mutate": 0 }
```

File: doc/EFFECT_ON_CONDITION.md [hunk 2]
```diff
@@ -172,3 +172,1082 @@ Other EOCs have some variables as well that they have access to, they are as fol
...
+```
+
+```json
+{ "npc_mutate_category": { "math": [ "1+1" ] }, "use_vitamins": false }
+```
+
+
+#### `u_mutate_category`, `npc_mutate_category`
+
+exactly same as u_mutate
```
Comment (andrei8l) [inline doc/EFFECT_ON_CONDITION.md:207]: This one takes mutation category as a parameter so it's not the same



```suggestion
Similar to `u_mutate` but takes category as a parameter and guarantees mutation.

| Syntax | Optionality | Value  | Info |
| --- | --- | --- | --- | 
| "u_mutate_category" / "npc_mutate_category" | **mandatory** | string or [variable object](#variable-object) | mutation category |
| "use_vitamins" | optional | boolean | same as in `u_mutate` | 

##### Examples
```json
{ "u_mutate_category": "PLANT" }
{ "u_mutate_category": { "global_val": "next_mutation" }
---->> github eats these backticks ``` <----

```

File: doc/EFFECT_ON_CONDITION.md [hunk 3]
```diff
@@ -172,3 +172,1082 @@ Other EOCs have some variables as well that they have access to, they are as fol
...
+```
+
+You increase the HP of your minor parts to 50, if possible
+```json
+{ "u_set_hp": 50, "minor_only": true, "only_increase": true }
+```
+
+You heal your right leg for 10 HP; in detail, you set the HP of your right leg to be 10 HP bigger than it's current HP; what people could do to not add `u_adjust_hp` XD
+```json
+{ "u_set_hp": { "math": { "u_val('hp', 'bodypart: leg_r')", "+", "10" } }, "target_part": "leg_r" }
```
Comment (andrei8l) [inline doc/EFFECT_ON_CONDITION.md:1227]:
```suggestion
{ "u_set_hp": { "math": { "u_val('hp', 'bodypart: leg_r') + 10" } }, "target_part": "leg_r" }
```

File: doc/EFFECT_ON_CONDITION.md [hunk 4]
```diff
@@ -172,3 +172,1097 @@ Other EOCs have some variables as well that they have access to, they are as fol
...
+
+##### Examples
+Adds `TELEPATH` trait to the character:
+```json
+{ "u_add_trait": "TELEPATH" }
+```
+
+Adds trait, stored in `trait_id` context value, to the character:
+```json
+{ "u_add_trait": { "context_val": "trait_id" }
```
Comment (bombasticSlacks) [inline doc/EFFECT_ON_CONDITION.md:337]:
```suggestion
{ "u_add_trait": { "context_val": "trait_id" } }
```

**Pull request merged**